src/spawn.ts

```diff
@@ -0,0 +1,29 @@
+import {
+  spawn as SpawnInternal,
+  SpawnOptionsWithStdioTuple,
+  StdioNull,
+  StdioPipe
+} from 'child_process'
+
+export type SpawnReturnArgs = {
+  exitCode: number | null
+}
+
+/**
+ * Simple wrapper around NodeJS's "child_process.spawn" function.
+ * Since we only use the exit code, we only expose that.
+ */
+export async function spawn(
```

Comment:
Can we rename this to `spawnAndWaitForExitCode()`? I find it a bit confusing that `spawn()` shouldn't return immediately upon spawning the process without waiting for it to finish, as that would be the semantics I've come to expect of any function called `spawn()`...